package.json

```diff
@@ -39,15 +39,19 @@
     "clsx": "^1.2.1",
     "date-fns": "^2.29.3",
     "graphql": "16.6.0",
+    "mkdirp-classic": "^0.5.3",
     "next": "^13.1.4",
     "next-pwa": "^5.6.0",
     "object-path": "^0.11.7",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.7.1",
+    "react-joyride": "^2.5.3",
     "react-responsive": "9.0.2",
     "react-swipeable": "6.1.2",
     "recharts": "^2.3.2",
+    "sharp": "^0.30.7",
+    "styled-components": "^4.4.1",
```

Comment:
Why do we need mkdirp-classic, sharp and styled-component ?

Reply:
thought it as dependency of react-tour but I removed.it's useless now i'll delete it